### PR TITLE
Changes for aucore-blockvote-11: Medication

### PR DESCRIPTION
**Pull request by dt-r**
This PR addresses issues from the AU Core R1 Ballot for Comment aucore-blockvote-11: https://confluence.hl7.org/display/HAFWG/aucore-blockvote-11

- Made the following changes to AU Core Medication:
  - removed Must Support from Medication.form [FHIR-45221](https://jira.hl7.org/browse/FHIR-45221)
  - removed Must Support from Medication.ingredient [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
  - removed Must Support from Medication.ingredient.item[x] [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
  - removed Must Support from Medication.ingredient.isActive [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220), [FHIR-45130](https://jira.hl7.org/browse/FHIR-45130), [FHIR-45049](https://jira.hl7.org/browse/FHIR-45049)
  - removed Must Support from Medication.ingredient.strength [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
  - removed Must Support from Medication.amount [FHIR-45219](https://jira.hl7.org/browse/FHIR-45219)
  - removal of Must Support from Medication.manufacturer [FHIR-45130](https://jira.hl7.org/browse/FHIR-45130).
- Updated 2 examples to remove elements that are no longer Must Support, noting left one Medication example with additional elements.
- Updated change log
- Updated general-requirements
- Updated profile-specific implementation guidance to remove guidance on manufacturer since that is no longer Must Support

### CODE REVIEW

**Review by dbojicic — COMMENTED** [review]
Changes to remove fragments for the elements that are not MS anymore are ok but the example id '1active-ingredient' is misleading now that the example only contains a medication code and no active ingredient information. Suggest changing the example id, name and description to something more suitable.

**Review by dbojicic — CHANGES_REQUESTED** [review]
A few minor comments to address.

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 
 - Removed Must Support from Location.description in AU Core Location [FHIR-45029](https://jira.hl7.org/browse/FHIR-45029).
+- Made the following changes to AU Core Medication:
+  - removed Must Support from Medication.form [FHIR-45221](https://jira.hl7.org/browse/FHIR-45221)
+  - removed Must Support from Medication.ingredient [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
+  - removed Must Support from Medication.ingredient.item[x] [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
+  - removed Must Support from Medication.ingredient.isActive [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220), [FHIR-45130](https://jira.hl7.org/browse/FHIR-45130), [FHIR-45049](https://jira.hl7.org/browse/FHIR-45049)
+  - removed Must Support from Medication.ingredient.strength [FHIR-45220](https://jira.hl7.org/browse/FHIR-45220)
+  - removed Must Support from Medication.amount [FHIR-45219](https://jira.hl7.org/browse/FHIR-45219)
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/pagecontent/changes.md:18]: I did intentionally leave those names for now. The medications still have that amount of active ingredients. I do expect that this content will be different post the example sweep that will change the majority of AU Core examples. 

No change.

**Comment by dt-r** [comment]
> Example changes to remove fragments for the elements that are not MS anymore are ok but the example ids '1active-ingredient' and '2active-ingredients' are misleading now that the example only contain a medication code and no active ingredient information. Suggest changing the example ids, names and descriptions to something more suitable.

I did intentionally leave those names for now. The medications still have that amount of active ingredients. I do expect that this content will be different post the example sweep that will change the majority of AU Core examples.

No change.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ This change log documents the significant updates and resolutions implemented fr
 - Added Security and Privacy page [FHIR-45067](https://jira.hl7.org/browse/FHIR-45067).
 - Removed Must Support from Location.description in AU Core Location [FHIR-45029](https://jira.hl7.org/browse/FHIR-45029).
 - Made the following changes to AU Core Medication:
+  - removal of Must Support from Medication.manufacturer FHIR-45130.
```
Comment (dbojicic) [inline input/pagecontent/changes.md:13]: Missing hyperlink to JIRA issue. 
Also has a full stop where others don't, making it inconsistent with other entries (noting a lack of the change log formatting conventions).

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please update terminology page to remove resources associated with the elements that are no longer Must Support.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: fixed both.

**Pull request merged**